test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -152,6 +170,151 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, prodClusterQ.Name, prodWl)
 			util.ExpectWorkloadsToBeWaiting(ctx, k8sClient, devWl)
 		})
+
+		ginkgo.It("Should allow for a workload to continue if the timeout is not exceeded", func() {
+			ginkgo.By("create the 'prod1' workload")
+			prodWl1 := testing.MakeWorkload("prod1", ns.Name).Queue(prodQueue.Name).Request(corev1.ResourceCPU, "2").Obj()
+			gomega.Expect(k8sClient.Create(ctx, prodWl1)).Should(gomega.Succeed())
+
+			ginkgo.By("create the 'prod2' workload after a second")
+			time.Sleep(time.Second)
```

Comment:
Alternatively, you could add priorities to the workloads to guarantee order and avoid sleeping.

Reply:
Used priorities now.